From 8198040e9579385275e11765e21b64e21f8403f6 Mon Sep 17 00:00:00 2001
From: sahandevs <sahandevs@gmail.com>
Date: Wed, 1 Feb 2023 16:46:21 +0330
Subject: [PATCH] b

---
 src/connection/connect.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/connection/connect.rs b/src/connection/connect.rs
index 9e38656..ad4d4e0 100644
--- a/src/connection/connect.rs
+++ b/src/connection/connect.rs
@@ -18,7 +18,7 @@ use tokio::{
 /// This connection can be used as a regular async TCP stream after replying the client.
 #[derive(Debug)]
 pub struct Connect<S> {
-    stream: TcpStream,
+    pub stream: TcpStream,
     _state: S,
 }
 
-- 
2.34.1

